### PR TITLE
add RLHF french translation

### PR DESCRIPTION
**Pull request by natolambert**

### CODE REVIEW

**Comment by lewtun** [comment]
Do you know if this is how we add translations for blog posts? In our other projects, we usually have a dedicated folder for translations & I'm wondering whether your approach here will render or not

cc @osanseviero who might have better ideas on how we handle blog post translations

**Comment by julien-c** [comment]
there is a `zh/` folder in this repo with translations to Chinese, i suggest you follow the same convention.

Note however that we don't have i18n of the blog beyond just EN and ZH

File: rlhf-fr.md [hunk 1]
```diff
@@ -0,0 +1,363 @@
+---
+title: "Illustrating Reinforcement Learning from Human Feedback (RLHF)" 
+thumbnail: /blog/assets/120_rlhf/thumbnail.png
+authors:
+- user: natolambert
+- user: LouisCastricato
+  guest: true
+- user: lvwerra
+- user: Dahoas
+  guest: true
```
Comment (julien-c) [inline rlhf-fr.md:10]: note that we have special metadata for translators, see this post: https://github.com/huggingface/blog/blob/main/zh/blip-2.md?plain=1

Comment (gultar) [reply rlhf-fr.md:10]: @julien-c 
Thanks for the heads up! I'll do just that.
If you are in need of French localization for the blog, let me know, and I'll take care of it.

**Comment by julien-c** [comment]
> How shall I submit the corrections to be made? Pull request?

Yes, can you open a new PR with the suggested changes @gultar?

**Comment by julien-c** [comment]
will close that one in the meantime

**Comment by antebachmann** [comment]
Hello, I am the original translator
I'd be happy to provide some extra proofreading and translations once my exams are over

**Comment by julien-c** [comment]
hi @antebachmann – great, please open a new PR! we'd love to merge it

**Comment by gultar** [comment]
@antebachmann 
Hi!
No problem. I'll open a pull request with recommanded changes to your translation, and terminological sources for you to refer to. Feel free to reach out if you have questions or comments about that.

**Comment by antebachmann** [comment]
Happily, and thanks for the help, I haven't yet translated a lot of academic text, so feedback is very much appreciated

**Pull request closed without merging**